sphinx/ext/autosectionlabel.py

```diff
@@ -29,43 +29,57 @@ def get_node_depth(node: Node) -> int:
     return i
 
 
-def get_node_parents(node: Node) -> list[str]:
+def _find_node_parents(node: Node) -> list[str]:
```

Comment:
I would revert this change because it could be used in the wild (and to make the diff smaller.

Reply:
Have renamed to _get_all_node_parent_section_titles to describe
more fully what the function does and _ prefix is just for internal use